homeassistant/components/deconz/config_flow.py

```diff
@@ -191,3 +196,45 @@ def __init__(self):
         user_input = {CONF_ALLOW_CLIP_SENSOR: True,
                       CONF_ALLOW_DECONZ_GROUPS: True}
         return await self.async_step_options(user_input=user_input)
+
+    async def async_step_hassio(self, user_input=None):
+        """Prepare configuration for a Hass.io deCONZ bridge.
+
+        This flow is triggered by the discovery component.
+        """
+        if configured_hosts(self.hass):
+            return self.async_abort(reason='one_instance_only')
```

Comment:
This won't work with multiple gateways, should check if the host is there.

Reply:
There is no support for multiple gateways yet, that's on a different PR (needing some review love) ;)